js/command-builder.js

```diff
@@ -153,13 +145,15 @@ export default class CommandBuilder {
           return res();
         }
         const effectUniforms = new Uniforms(globalId);
+        const effectAttributes = new Attributes(globalId);
         const effectClass = effectConfig.getEffectClass();
         vert.mainBody += `if (${effectConfig.timeBegin} <= globalTime && globalTime <= ${effectConfig.timeEnd}) {`;
-        effectClass.registerAsync(effectConfig, this.props, effectUniforms, vert)
+        effectClass.registerAsync(effectConfig, this.props, effectUniforms, vert, frag, effectAttributes)
```

Comment:
Passing the fragment shader into `register` was of course not necessary. But I believed it was in the beginning, so now it's there and I don't think it's a bad thing. But I can of course revert it